hydra/test_utils/launcher_common_tests.py

```diff
@@ -350,6 +358,18 @@ def sweep_two_config_groups(
 
 @pytest.mark.usefixtures("restore_singletons")
 class IntegrationTestSuite:
+    def custom_overrides(self) -> List[str]:
+        return []
+
+    def temp_remote_dir(self) -> Optional[Path]:
+        return None
+
+    def download(self) -> Optional[Callable[[str, str], None]]:
+        return None
+
+    def working_dir(self) -> Optional[Path]:
+        return None
+
```

Comment:
Add a few comments describing who should use this class, how, and are each of the methods.

Does this make sense for other remote launchers?

Reply:
resolving, these all gone